src/shared/directives/conditional-show/domain/conditional-show.model.spec.ts

```diff
@@ -548,7 +549,7 @@ describe('conditional-show', () => {
     it('should return true when all conditions are true when using Complex fields', () => {
       caseField1.value = ['s1', 's2', 's3'];
       let sc = new ShowCondition(
-        'field1CONTAINS"s3,s2" AND claimantDetails.AddressUKCode.PostTown="London" AND claimantDetails.AddressUKCode.Country="UK"');
+        'field1 CONTAINS"s3,s2" AND claimantDetails.AddressUKCode.PostTown="London" AND claimantDetails.AddressUKCode.Country="UK"');
```

Comment:
Same as the comment above about operators not needing to have spaces before or after...

Reply:
As I mentioned in my earlier comment